OneSignalSDK/onesignal/core/src/test/java/com/onesignal/session/internal/outcomes/OutcomeEventsRepositoryTests.kt

```diff
@@ -171,13 +171,15 @@ class OutcomeEventsRepositoryTests : FunSpec({
                     OutcomeEventsTable.COLUMN_NAME_NAME to "outcomeId1",
                     OutcomeEventsTable.COLUMN_NAME_WEIGHT to 0.2f,
                     OutcomeEventsTable.COLUMN_NAME_TIMESTAMP to 1111L,
+                    OutcomeEventsTable.COLUMN_NAME_SESSION_TIME to 1L,
```

Comment:
Noting this column was added in https://github.com/OneSignal/OneSignal-Android-SDK/pull/1793